bundler/spec/dependabot/bundler/file_parser_spec.rb

```diff
@@ -373,14 +335,7 @@
       end
 
       context "that comes from a .specification file" do
-        let(:files) { [gemfile, lockfile, specification] }
-        let(:specification) do
-          Dependabot::DependencyFile.new(
-            name: "plugins/example/.specification",
-            content: fixture("ruby", "specifications", "statesman"),
-            support_file: true
-          )
-        end
+        let(:dependency_files) { project_dependency_files("bundler1/version_specified_gemfile_specification") }
```

Comment:
does this not need the same `support_file = true` setup?

Reply:
It might? I thought all tests were passing locally but it seems I may have missed one or two, this might be one of them!